### PR TITLE
docs(zh-cn): dont-block-the-event-loop.md mailformed link

### LINKED ISSUE #5230: A section of the guide Don't Block the Event Loop (or the Worker Pool) is not displayed correctly
### URL:

https://nodejs.org/zh-cn/docs/guides/dont-block-the-event-loop#%E5%A6%82%E4%BD%95%E8%BF%9B%E8%A1%8C%E4%BB%BB%E5%8A%A1%E5%88%86%E6%B5%81

### Browser Name:

Google Chrome

### Browser Version:

111.0.5563.147

### Operating System:

Windows 10

### How to reproduce the issue:

A section of the guide Don't Block the Event Loop (or the Worker Pool) is not displayed correctly.

Screenshots:

### PR DESCRIPTION
**Pull request by nschonni**
Fixes #5230
Should probably go through Crowdin, but it looks like these links got mangled. Not sure if it was with their editor or on the export

### CODE REVIEW

**Comment by bnb** [comment]
might be worth putting up [https://github.com/JustinBeckwith/linkinator](https://github.com/JustinBeckwith/linkinator) to find all instances of broken links on the website. There's a GitHub Action that I set up in the OpenJS Foundation CPC repo that does it as a check, if that's helpful.

**Review by bnb — APPROVED** [review]

**Comment by nschonni** [comment]
I think Linkinator used to be setup here (and might still be), but the problem here is around mailformed links, so Linkinator wouldn't catch this. Markdownlint or remark-lint would catch that stuff, but I'm not sure their state since the migration

File: pages/zh-cn/docs/guides/dont-block-the-event-loop.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ asyncAvg(n, function(avg){
 1. 你可以通过开发 [C++ 插件](https://nodejs.org/api/addons.html) 的方式使用内置的 Node.js 工作池。稍早之前的 Node.js 版本，通过使用 [NAN](https://github.com/nodejs/nan) 的方式编译你的 C++ 插件，在新版的 Node.js 上使用 [N-API](https://nodejs.org/api/n-api.html)。 [node-webworker-threads](https://www.npmjs.com/package/webworker-threads) 提供了一个仅用 JavaScript 就可以访问 Node.js 的工作池的方式。
 2. 您可以创建和管理自己专用于计算的工作线程池，而不是使用 Node.js 自带的负责的 I/O 的工作线程池。最直接的方法就是使用 [Child Process](https://nodejs.org/api/child_process.html) 或者是 [cluster](https://nodejs.org/api/cluster.html)。
 
-你 *不应该*直接为每个请求都创建一个[子进程](https://nodejs.org/api/child_process.html)[。 因为客户端请求的频率可能远远高于你的服务器能创建和管理子进程的频率，这种情况你的服务器就变成了一个<](https://en.wikipedia.org/wiki/Fork_bomb)a href="https://en.wikipedia.org/wiki/Fork_bomb">Fork 炸弹</a>。
+你 *不应该*直接为每个请求都创建一个[子进程](https://nodejs.org/api/child_process.html)。 因为客户端请求的频率可能远远高于你的服务器能创建和管理子进程的频率，这种情况你的服务器就变成了一个[炸弹](https://en.wikipedia.org/wiki/Fork_bomb)。
```
Comment (nschonni) [inline pages/zh-cn/docs/guides/dont-block-the-event-loop.md]: Not sure if "Fork" is actually still supposed to be in the label
```suggestion
你 *不应该*直接为每个请求都创建一个[子进程](https://nodejs.org/api/child_process.html)。 因为客户端请求的频率可能远远高于你的服务器能创建和管理子进程的频率，这种情况你的服务器就变成了一个[Fork 炸弹](https://en.wikipedia.org/wiki/Fork_bomb)。
```

**Review by ghost — APPROVED** [review]
Thank you @nschonni , and this is what I mentioned with @ovflowd on the formation of Crowdin's error, no ideas on that. Though I translate it one by one, maybe I'll have a discussion in the community of Crowdin about that....

**Comment by nschonni** [comment]
Feel free to fix this on the Crowdin side instead @MaledongGit , I just didn't think it made sense to try to join the translators group for a language I don't know to fix a formatting issue 😆

**Review by aymen94 — APPROVED** [review]

**Comment by ovflowd** [comment]
Let's fix this issue on Crowdin side if possible 👀 (Feel free to reopen if we cannot fix this issue from there, @MaledongGit)

> Thank you @nschonni , and this is what I mentioned with @ovflowd on the formation of Crowdin's error, no ideas on that. Though I translate it one by one, maybe I'll have a discussion in the community of Crowdin about that....

Yeah, let's open an discussion there and a support ticket. But @nschonni or @bnb or @MaledongGit feel free to add a Linkinator step on our "Lint" GitHub Action Workflow.

**Comment by ghost** [comment]
> Let's fix this issue on Crowdin side if possible 👀 (Feel free to reopen if we cannot fix this issue from there, @MaledongGit)
> 
> > Thank you @nschonni , and this is what I mentioned with @ovflowd on the formation of Crowdin's error, no ideas on that. Though I translate it one by one, maybe I'll have a discussion in the community of Crowdin about that....
> 
> Yeah, let's open an discussion there and a support ticket. But @nschonni or @bnb or @MaledongGit feel free to add a Linkinator step on our "Lint" GitHub Action Workflow.

OK, I've re-submitted a change in the Crowdin to see whether there'll be any changes to fix it, if not, I think I have to re-open it and them merge, finally copy the link address directly into Crowdin's forum help...

Thanks all!

**Pull request closed without merging**